spk/shairport/Makefile

```diff
@@ -0,0 +1,24 @@
+SPK_NAME = shairport
+SPK_VERS = 1.1.1
+SPK_REV = 2
+SPK_ICON = src/shairport.png
+
+DEPENDS =  cross/$(SPK_NAME)
+
+MAINTAINER = Lost Carrier
+DESCRIPTION = Use Synology NAS as an AirPlay-Receiver. Requires USB-Soundcard or speakers.
+RELOAD_UI = yes
```

Comment:
I believe this is not needed if there's no UI interaction (e.g. no icon to be placed)